vulcanai2/tests/models/test_cnn.py

```diff
@@ -136,5 +139,19 @@ def test_fit_multi_input(self, multi_input_cnn,
                           range(len(multi_input_cnn_data)//2, 
                                     len(multi_input_cnn_data)))
         test_train_loader = DataLoader(test_train, batch_size=2)
-        test_val_loader = DataLoader(test_val, batch_size=2)
-        multi_input_cnn.fit(test_train_loader, test_val_loader, 1)
\ No newline at end of file
+        test_val_loader = DataLoader(test_val, batch_size=2)     
+        
+        init_weights = copy.deepcopy(multi_input_cnn.network[0]._kernel.weight.data)
+        multi_input_cnn_no_fit = copy.deepcopy(multi_input_cnn)
+        try:
+            multi_input_cnn.fit(test_train_loader, test_val_loader, 2)
+        except RuntimeError:
+            print("The network multi_input_cnn failed to train.")
```

Comment:
please use the logger instead of printing

Reply:
you just have to write `logger.error("string")`. see the other files for more examples. I'd like this to be done before merging